### PR TITLE
Add Nucleoid.xyz

### PR DESCRIPTION
**Pull request by GeefireMC**
### General:
* [X] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated metadata.json`)*
* [X] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [X] Folder with appropriate name has been created / updated (must not include any non-alpha characters or whitespace).
* [X] No changes were made to the file's formatting (4 spaces per tab).
* [X] There are no syntax errors.
* [X] There is no pre-existing mapping matching my id (check if there is an existing folder).
* [X] My field values match your requirements.
* You can view our patterns here: [metadata.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/metadata.schema.json), or take a look below and complete the field checklist:
  - [X] `id`: a lowercase string, which should match the folder name *(ex. `myserver`)*
  - [X] `name`: a string *(ex. `MyServerPvP`)*
  - [X] `addresses`: an array with lowercase strings *(ex. `["my.server", "your-server.com"]`)*
    - You do not need to specify subdomains, Lunar Client services automatically detect them.
  - [X] `primaryAddress`: an address *(ex. `my.server`)*
  - [X] `minecraftVersions`: an array with Minecraft versions as strings *(ex. `["1.18", "1.19"]` - Must be versions supported by Lunar Client)*
  - [X] `primaryMinecraftVersion`: a Minecraft version as a string *(ex. `1.18` - Must be versions supported by Lunar Client)*
  - [ ] `primaryColor`: a hexademical color code that primarily distiguishes the server *(ex. `#00FFFF`)* 
  - [ ] `secondaryColor`: a hexademical color code that accompanies the `primaryColor` of the server *(ex. `#FF0000`)*
  - [X] `primaryRegion`: the primary region where your server operates in *(ex. `NA`)*
  - [X] `regions`: a list of regions where you have servers located that service your players *(ex. `["NA", "EU", "AS"]`)*
  - [X] `gameTypes`: a list of games that describe the content on your server, must be a max of 3 listed *(ez. `["PVP", "UHC", "HCF"]`)*

#### Logo
* [X] My logo is a `png` file.
* [X] I have uploaded my logo to my server folder *(ex. `lunarnetwork`)* and named it `logo.png`.
* [X] My logo has a transparent background and is square (1:1 aspect ratio).
* [X] My logo is `512` pixels in width and height.
* [ ] My logo is my own property and complies with relevant copyright/privacy laws.

#### Background
* [X] My background is a `png` file.
* [X] I have uploaded my background to my server folder *(ex. `lunarnetwork`)* and named it `background.png`.
* [X] My background has an aspect ratio of 16:9.
* [X] My background is atleast `1920` pixels in width and `1080` pixels in height (can be bigger, but must remain the same aspect ratio).
* [X] My background contains relevant content pertaining to the server, is my own property, and complies with relevant copyright/privacy laws.
* [X] My background doesn't contain any markings, text, or logos (unless part of a build).

### CODE REVIEW

**Review by itsJordan — CHANGES_REQUESTED** [review]
Please add the missing primaryColor and secondaryColor fields.

**Comment by GeefireMC** [comment]
> Please add the missing primaryColor and secondaryColor fields.

what are the primaryColor and secondaryColor fields used for? The server doesn't really have any specific "official" colors afaik, so knowing what they're used for would allow me to choose some nicer looking colors.

File: servers/nucleoid/metadata.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "id": "nucleoid",
+    "name": "Nucleoid",
+    "addresses": [
+        "nucleoid.xyz",
```
Comment (itsJordan) [inline servers/nucleoid/metadata.json:5]:
```suggestion
        "nucleoid.xyz"
```

File: servers/nucleoid/metadata.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "primaryLanguage": "en",
+    "languages": [
+        "en"
+    ],
+    "primaryRegion": "EU",    
+    "regions": [
+        "EU",
+        "NA"
+    ],
+    "gametypes": [
```
Comment (itsJordan) [inline servers/nucleoid/metadata.json:24]:
```suggestion
    "gameTypes": [
```

File: servers/nucleoid/metadata.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "languages": [
+        "en"
+    ],
+    "primaryRegion": "EU",    
+    "regions": [
+        "EU",
+        "NA"
+    ],
+    "gameTypes": [
+        "PVP",
```
Comment (itsJordan) [inline servers/nucleoid/metadata.json:25]:
```suggestion
        "PvP",
```

**Review by itsJordan — APPROVED** [review]

**Pull request merged**